test/decoders/test_video_decoder.py

```diff
@@ -365,6 +401,47 @@ def test_get_frame_displayed_at_fails(self):
         with pytest.raises(IndexError, match="Invalid pts in seconds"):
             frame = decoder.get_frame_displayed_at(100.0)  # noqa
 
+    def test_get_frames_displayed_at(self):
+
+        decoder = VideoDecoder(NASA_VIDEO.path)
+        ref_frame6 = NASA_VIDEO.get_frame_by_name("time6.000000")
```

Comment:
I prefer that we use the testing utilities rather than hard-coding these values. What I did when testing pts-based APIs is that I still compared them to frames I got by index in the testing framework - we know what frame it's going to be, and that alleviates the testing framework from also supporting pts-based access.

It's difficult to verify hard-coded values, and it's particularly difficult to know if the same hard-coded values are supposed to represent the same actual thing. The framework has these values in one place, and also explains how we got the values, and potentially how we would update them if we ever need to.